Index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="Style.css">
+    <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
+</head>
+<body>
+
+ <header class="top-bar">
+  <a href="#" id="logolink"><h2 class="logoname">My Logo</h2></a>
+  <nav >
+    <img src="images\Union.png" alt="logo">
+  </nav>
+</header>
+ <section class="section-data">
+    <h1 class="myname">I’m Surya Glad to see you!</h1>
+    <p class="namedetail">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <div>
+      <h3 class="connect">LET’S CONNECT</h3>
+        <ul class="socialmedia">
+          <li><a href="#"><img class="twitter" src="images\twitter.png" alt="twitter"></a></li>
+          <li><a href="#"><img class="dot" src="images\dot.png" alt="dot"></a></li>
+          <li><a href="#"><img class="i" src="images\i.png" alt="i"></a></li>
+          <li><a href="#"><img class="n" src="images\n.png" alt="n"></a></li>
+          <li><a href="#"><img class="m" src="images\M.png" alt="m"></a></li>
+          <li><a href="#"><img class="g" src="images\g.png" alt="g"></a></li>
+          <li><a href="#"><img class="v" src="images\v.png" alt="v"></a></li>
+        </ul>
```

Comment:
As suggested by the previous reviewer, kindly make sure the margins of the images is similar to that of the figma design. Kindly check the figma designs for the dimensions since there are some differences between yours. 
Also, it looks lilke you applied `display: flex` to the container of the images but it is not working well. You don't need to state the `position, top, right, left, and bottom` values for each of the social media icons when the `display: flex` on the containing element is working well
Please check and ensure the margins and the flex display are according to the figma design to help you achieve a more proffesional  project. :wink: 


### Your design 

![Screenshot from 2023-02-17 08-35-10](https://user-images.githubusercontent.com/65233860/219594074-ad26c608-1aeb-4dfe-adaa-7e7dc13b322a.png)

### Your design when `list-style-type` is displayed

![Screenshot from 2023-02-17 08-03-54](https://user-images.githubusercontent.com/65233860/219594587-ad89c29b-ff45-4169-ba27-a9064d426375.png)


### Figma design

![Screenshot from 2023-02-17 08-03-32](https://user-images.githubusercontent.com/65233860/219594195-60ba053a-67a7-426c-bd5e-6ed78cbb5a5e.png)